providers/os/connection/snapshot/blockdevices.go

```diff
@@ -163,7 +163,7 @@ func (blockEntries BlockDevices) FindDevice(requested string) (BlockDevice, erro
 }
 
 // Searches all the partitions in the device and finds one that can be mounted. It must be unmounted, non-boot partition
-func (device BlockDevice) GetMountablePartitions(includeAll bool) ([]*PartitionInfo, error) {
+func (device BlockDevice) GetMountablePartitions(includeBoot bool, includeMounted bool) ([]*PartitionInfo, error) {
```

Comment:
nit (with the latest changes): 
```suggestion
func (device BlockDevice) GetPartitions(includeBoot bool, includeMounted bool) ([]*PartitionInfo, error) {
```